guides/pyart_cheatsheet.tex

```diff
@@ -123,7 +121,7 @@
 need to get the latest master branch from github.com\\
 \\
 \-\hspace{0.1cm} $\bullet$ Clone the Py-ART repository:\\
-\-\hspace{0.1cm} \$ git clone https://github.com/ARM-DOE/pyart.git\\
+\-\hspace{0.1cm} \$ git clone git@github.com:ARM-DOE/pyart.git\\
```

Comment:
What's the reason for using ssh instead of http?

Reply:
git clone https:// I believe was discontinued I believed

Reply:
its the first option that shows up for m
<img width="671" alt="Screen Shot 2022-08-10 at 3 59 38 PM" src="https://user-images.githubusercontent.com/26660300/184019046-df76bca3-f577-4b9d-8e9a-6ff29ecf4beb.png">
e

Reply:
Hmm maybe to  push changes to a repo it has to be ssh? I thought they changed the whole cloning pushing syntax. I can revert back.

Reply:
Okay! Thanks - yeah, I think its also personal preference... it's also the default which is why I usually use it